docs/literate/src/files/hohqmesh_literate.jl

```diff
@@ -0,0 +1,521 @@
+#    \end{END_POINTS_LINE}
+#
+#    \begin{END_POINTS_LINE}
+#       name = RightSlant
+#       xStart = [ 0.0, -3.0, 0.0]
+#       xEnd   = [ 2.0, 1.0, 0.0]
+#    \end{END_POINTS_LINE}
+#
+#    \begin{CIRCULAR_ARC}
+#       name        = IceCream
+#       units       = degrees
+#       center      = [ 0.0, 1.0, 0.0]
+#       radius      = 2.0
+#       start angle = 0.0
+#       end angle   = 180.0
+#    \end{CIRCULAR_ARC}
+#    \end{CHAIN}
+#
+# \end{INNER_BOUNDARIES}
+# ```
+
+# It is important to note there are two `name` quantities one for the `CHAIN` and one for the `PARAMETRIC_EQUATION_CURVE`.
+# The name for the `CHAIN` is used internally by HOHQMesh, so if you have multiple `CHAIN`s they **must be given a unique name**.
+# The name for the `PARAMETRIC_EQUATION_CURVE` will be printed to the appropriate boundaries within the `.mesh` file produced by
+# HOHQMesh. The resulting mesh generated by HOHQMesh.jl is given in the following figure.
+
+# ![mesh_straight_sides](https://user-images.githubusercontent.com/25242486/129603958-08e4b874-53d5-4511-9a54-6daf4c21edca.png)
+
+# The mesh file `ice_cream_straight_sides.mesh` and its associated file for plotting
+# `ice_cream_straight_sides.tec` are placed in the `out` folder.
+# We note that Trixi.jl uses the boundary name information from the control file
+# to assign boundary conditions in an elixir file.
+# Therefore, the name should start with a letter and consist only of alphanumeric characters and underscores. Please note that the name will be treated as case sensitive.
+
+# ## Example simulation on `ice_cream_straight_sides.mesh`
+
+# With this newly generated mesh we are ready to run a Trixi.jl simulation on an unstructured quadrilateral mesh.
+# For this we must create a new elixir file.
+
+# The elixir file given below creates an initial condition for a
+# uniform background flow state with a free stream Mach number of 0.3.
+# A focus for this part of the tutorial is to specify the boundary conditions and to construct the new mesh from the
+# file that was generated in the previous exercise.
+
+# It is straightforward to set the different boundary
+# condition types in an elixir by assigning a particular function to a boundary name inside a
+# Julia dictionary, `Dict`, variable. Observe that the names of these boundaries match those provided by HOHQMesh
+# either by default, e.g. `Bottom`, or user assigned, e.g. `IceCream`. For this problem setup use
+# * Freestream boundary conditions on the four box edges.
+# * Free slip wall boundary condition on the interior curved boundaries.
+
+# We create the mesh file `ice_cream_straight_sides.mesh` and its associated file for plotting
+# `ice_cream_straight_sides.tec` by using HOHQMesh's function `generate_mesh`.
+using HOHQMesh
+control_file = joinpath(@__DIR__, "..", "ice_cream_straight_sides.control")
+control_file = joinpath("docs", "src", "ice_cream_straight_sides.control") #src
+output = generate_mesh(control_file);
```

Comment:
I added this part to the tutorial to make it work (also for curved sides).
Maybe I need to update the text accordingly (see above: "The mesh file `ice_cream_straight_sides.mesh` and its associated file for plotting `ice_cream_straight_sides.tec` are placed in the `out` folder.").

Reply:
Does this tutorial use any real code that you run, too? Or is it more or less a text to follow?

Reply:
Yes, please